tpm2/tpm2.go

```diff
@@ -1838,3 +1838,44 @@ func RSADecrypt(rw io.ReadWriter, key tpmutil.Handle, password string, message [
 	}
 	return decodeRSADecrypt(resp)
 }
+
+// DictionaryAttackLockReset cancels the effect of a TPM lockout due to a number of successive authorization failures.
+// If this command is properly authorized, the lockout counter is set to zero.
+func DictionaryAttackLockReset(rw io.ReadWriter, auth AuthCommand) error {
+	ha, err := tpmutil.Pack(HandleLockout)
+	if err != nil {
+		return err
+	}
+	encodedAuth, err := encodeAuthArea(auth)
+	if err != nil {
+		return err
+	}
+	cmd, err := concat(ha, encodedAuth)
+	if err != nil {
+		return err
+	}
+	_, err = runCommand(rw, TagSessions, cmdDictionaryAttackLockReset, tpmutil.RawBytes(cmd))
+	return err
+}
+
+// DictionaryAttackParameters changes the lockout parameters
```

Comment:
Make this a full sentence, and maybe describe what kind of authorization is typically needed?

Reply:
If we describe the auth concerns in `DictionaryAttackLockReset`, we can just say that the auth for this command is the same, and leave it at that.